backend/init/init.go

```diff
@@ -47,6 +48,7 @@ func init() {
 			`Warning: "azure" name is deprecated, please use "azurerm"`),
 		"azurerm": func() backend.Backend { return backendAzure.New() },
 		"etcdv3":  func() backend.Backend { return backendetcdv3.New() },
+		"manta":   func() backend.Backend { return backendManta.New() },
```

Comment:
For historical reasons this was called Manta but arguably that's an internal name for the project now. From the product side of things its Triton Object Store (hence the reason `github.com/joyent/triton-go/storage`). I'm wondering if we should rename this to Triton and deprecate the Manta brand from TF completely.

Thoughts?

Reply:
Just to make it known, @cheapRoc and I discussed this internally and decided to leave it as manta :)